static/app/views/settings/organizationMembers/inviteBanner.tsx

```diff
@@ -0,0 +1,219 @@
+        <CardTitleContent>
+          <CardTitle>{t('Bring your full GitHub team on board in Sentry')}</CardTitle>
+          <Subtitle>
+            {tct('[missingMemberCount] missing members that are active in your GitHub', {
+              missingMemberCount: users.length,
+            })}
+            <Tooltip title="Based on the last 30 days of commit data">
+              <IconInfo size="xs" />
+            </Tooltip>
+          </Subtitle>
+        </CardTitleContent>
+        <ButtonContainer>
+          <Button
+            priority="primary"
+            size="xs"
+            // TODO(cathy): open up invite modal
+            data-test-id="view-all-missing-members"
+          >
+            {t('View All')}
+          </Button>
+        </ButtonContainer>
+      </CardTitleContainer>
+      <MemberCardsContainer>{cards}</MemberCardsContainer>
+    </StyledCard>
+  );
+}
+
+export default withOrganization(InviteBanner);
+
+type SeeMoreCardProps = {
+  missingUsers: MissingMember[];
+};
+
+function SeeMoreCard({missingUsers}: SeeMoreCardProps) {
+  return (
+    <MemberCard data-test-id="see-more-card">
+      <MemberCardContent>
+        <MemberCardContentRow>
+          <SeeMore>
+            {tct('See all [missingMembersCount] missing members', {
+              missingMembersCount: missingUsers.length,
+            })}
+          </SeeMore>
+        </MemberCardContentRow>
+        <Subtitle>
+          {tct('Accounting for [totalCommits] recent commits', {
+            totalCommits: missingUsers.reduce((acc, curr) => acc + curr.commitCount, 0),
+          })}
+        </Subtitle>
+      </MemberCardContent>
+      <Button
+        size="sm"
+        priority="primary"
+        // TODO(cathy): open up invite modal
+        data-test-id="view-all-missing-members"
+      >
+        {t('View All')}
+      </Button>
+    </MemberCard>
+  );
+}
+
+const StyledCard = styled(Card)`
+  padding: ${space(2)};
+  display: flex;
+  overflow: hidden;
+`;
+
+const CardTitleContainer = styled('div')`
+  display: flex;
+  justify-content: space-between;
+`;
+
+const CardTitleContent = styled('div')`
+  display: flex;
+  flex-direction: column;
+`;
+
+const CardTitle = styled('div')`
+  font-size: ${p => p.theme.fontSizeLarge};
+  font-weight: bold;
+  color: ${p => p.theme.gray400};
+`;
+
+const Subtitle = styled('div')`
+  font-size: ${p => p.theme.fontSizeSmall};
+  font-weight: 400;
+  color: ${p => p.theme.gray300};
+  display: flex;
+  align-items: center;
+  & > *:first-child {
+    margin-left: ${space(0.5)};
+    display: flex;
+    align-items: center;
+  }
+`;
+
+const ButtonContainer = styled('div')`
+  display: grid;
+  grid-auto-flow: column;
+  grid-column-gap: ${space(1)};
+`;
+
+const MemberCard = styled(Card)`
+  padding: ${space(2)} 18px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin: ${space(1)} ${space(0.5)} 0 0;
+  min-width: 30%;
+  justify-content: center;
+  flex-wrap: wrap;
```

Comment:
Nit: I like to sort my CSS in this order based on semantics so it's easier to visualized the element.

1. Page properties (`position`, `top`, `left`, etc)
2. Block properties (`display`, `flex-etc`, `width`)
3. Box properties (`margin`, `padding`, `border`, `content`)
4. "Inside the box" properties (`font`, `overflow`, etc)